### PR TITLE
Cnn

### PR DESCRIPTION
**Pull request by AKAazure**

### CODE REVIEW

File: examiner-baseline.ipynb [hunk 1]
```diff
@@ -0,0 +1 @@
...torch.cat([i for i in res]).float().mean())\n        ","metadata":{"trusted":true},"execution_count":null,"outputs":[]},{"cell_type":"code","source":"","metadata":{},"execution_count":null,"outputs":[]},{"cell_type":"code","source":"","metadata":{},"execution_count":null,"outputs":[]},{"cell_type":"code","source":"","metadata":{},"execution_count":null,"outputs":[]},{"cell_type":"code","source":"","metadata":{},"execution_count":null,"outputs":[]},{"cell_type":"code","source":"","metadata":{},"execution_count":null,"outputs":[]},{"cell_type":"code","source":"","metadata":{},"execution_count":null,"outputs":[]},{"cell_type":"code","source":"","metadata":{},"execution_count":null,"outputs":[]},{"cell_type":"code","source":"","metadata":{},"execution_count":null,"outputs":[]},{"cell_type":"code","source":"","metadata":{},"execution_count":null,"outputs":[]},{"cell_type":"code","source":"","metadata":{},"execution_count":null,"outputs":[]},{"cell_type":"code","source":"","metadata":{},"execution_count":null,"outputs":[]},{"cell_type":"code","source":"","metadata":{},"execution_count":null,"outputs":[]},{"cell_type":"code","source":"","metadata":{},"execution_count":null,"outputs":[]}]}
```
Comment (amanichopra) [inline examiner-baseline.ipynb:1]: What is this notebook for?

File: cnn_env.yaml
Comment (amanichopra) [inline cnn_env.yaml:1]: And also whats this file for?

File: examiner-baseline.ipynb (hunk 1 above)
Comment (AKAazure) [reply examiner-baseline.ipynb:1]: This is the notebook exported from Kaggle. It does not really matter, and it is bascially the same as the .py file.

File: cnn_env.yaml
Comment (AKAazure) [reply cnn_env.yaml:1]: This is the conda environment configuration including information of library dependencies.

**Pull request merged**